backend/lib/services/members/MemberManager.js

```diff
@@ -187,41 +174,24 @@ class MemberManager {
     this.subjectList.delete(item.id)
   }
 
-  async deleteServiceAccountSecrets (item) {
-    const { namespace } = Member.parseUsername(item.id)
-    if (namespace !== this.namespace) {
-      throw new UnprocessableEntity('It is not possible to modify a ServiceAccount from another namespace')
-    }
-
-    const results = await _
-      .chain(item)
-      .get('extensions.secrets')
-      .map(({ name }) => this.client.core.secrets.delete(namespace, name))
-      .thru(promises => Promise.allSettled(promises))
-      .value()
-
-    this.constructor.getFulfilledValues(results)
-  }
-
   async getKubeconfig (item) {
-    const { namespace, name } = Member.parseUsername(item.id)
+    const defaultTokenExpiration = _.get(config, 'frontend.serviceAccountDefaultTokenExpiration', 7776000) // default is 90 days
```

Comment:
Why do we define the default in the code and not in the chart? Do we need a default in our deployment at all and let the this empty per default. This would mean we rely on max value of the apiserver.